src/libs/actions/Policy/Policy.ts

```diff
@@ -285,8 +286,18 @@ function isCurrencySupportedForDirectReimbursement(currency: string) {
 /**
  * Checks if the currency is supported for global reimbursement
  */
-function isCurrencySupportedForGlobalReimbursement(currency: TupleToUnion<typeof CONST.DIRECT_REIMBURSEMENT_CURRENCIES>, canUseGlobalReimbursementsOnND: boolean) {
-    return canUseGlobalReimbursementsOnND ? CONST.DIRECT_REIMBURSEMENT_CURRENCIES.includes(currency) : currency === CONST.CURRENCY.USD;
+function isCurrencySupportedForGlobalReimbursement(
```

Comment:
Curious - why is this function doing two things - checking for global reimbursements or whether Expensify Cards UK/EU can be used? In any case, the function name and doc doesn't reflect the updates here